crema/parsers/crux.py

```diff
@@ -94,5 +109,111 @@ def _parse_psms(txt_file, cols):
     pandas.DataFrame
         A :py:class:`pandas.DataFrame` containing the parsed PSMs
     """
-    LOGGER.info("Reading PSMs from %s...", txt_file)
-    return pd.read_csv(txt_file, sep="\t", usecols=cols)
+    if log:
+        LOGGER.info("Reading PSMs from %s...", txt_file)
+    return pd.read_csv(txt_file, sep="\t", usecols=lambda c: c in cols)
+
+
+def _create_pairing(pairing_data, pairing_fields):
+    """Parse a single Crux tab-delimited file
+
+    Parameters
+    ----------
+    pairing_data : pandas.DataFrame
+        A collection of PSMs with the necessary columns to create a target/decoy peptide pairing.
+        Required columns are "peptide mass", "sequence", "target/decoy", "original target sequence"
+
+    Returns
+    -------
+    pairing : dict
+        A map of target and decoy peptide sequence pairings
+    """
+    # ensure pairing_data dataframe contains all necessary columns
+    for field in pairing_fields:
```

Comment:
That's a good question. I returned None in the case that I couldn't generate a pairing, but I think since crux has a standard file format, it might be good to just raise an error since I should be able to generate a pairing given a crux file. Yes?